### PR TITLE
gremlin-test take 2

### PR DESCRIPTION
**Pull request by Fitzoh**
Follow up to  #65.
Adds in `lein-junit` plugin and allows the ability to run the `gremlin-test` suite instead of/in addition to maintaining a separate set of tests for `Ogre`.
the `clojurewerkz.ogre.suite.has-test` ns provides the traversals required for use by `OgreHasTest.Traversals`.

Those traversals are then used by then tested by `gremlin-test`

current `lein junit` status:
```
Testsuite: org.clojurewerkz.ogre.gremlin.process.OgreTinkerGraphProcessStandardTest
Tests run: 22, Failures: 10, Errors: 9, Time elapsed: 1.928 sec
```

### CODE REVIEW

File: test/clojure/clojurewerkz/ogre/suite/has_test.clj [hunk 1]
```diff
@@ -0,0 +1,94 @@
+(ns clojurewerkz.ogre.suite.has-test
+  (:require [clojure.test :refer [deftest testing is]]
+            [clojurewerkz.ogre.core :as q]
+            [clojurewerkz.ogre.vertex :as v]
+            [clojurewerkz.ogre.test-util :as u]))
+
+(defn get_g_V_outXcreatedX_hasXname__mapXlengthX_isXgtX3XXX_name [g]
```
Comment (michaelklishin) [inline test/clojure/clojurewerkz/ogre/suite/has_test.clj]: Please follow the code style ClojureWerkz projects have. Argument vector should be on the line after `defn`.

**Comment by michaelklishin** [comment]
I'm going to merge this because it's WIP anyway but please follow the code style per my comment.

**Comment by spmallette** [comment]
:+1:

File: test/clojure/clojurewerkz/ogre/suite/has_test.clj (hunk 1 above)
Comment (Fitzoh) [inline test/clojure/clojurewerkz/ogre/suite/has_test.clj]: Noted. Is there a style guide that exists somewhere?

**Pull request merged**